src/tactic/ui/widget/calendar_wdg.py

```diff
@@ -906,7 +912,11 @@ def get_display(my):
             text = input
         else:
             # TODO: add a kwarg - hint_text
-            input = TextInputWdg( name=name, read_only=read_only, required=required, icon=activator, width=width)
+            if my.get_option('hint_text'):
+                hint_text = my.get_option('hint_text')
+            else:
+                hint_text = None
```

Comment:
Try something like this instead:

hint_text = my.get_option('hint_text')
if not hint_text:
    hint_text = None